defaults/main/exec.yml

```diff
@@ -0,0 +1,16 @@
+---
+collectd_plugin_exec_exec: []
+# collectd_plugin_exec_exec:
+# - user: collectd
+#   group: collectd
+#   exec: "my-prog"
+#   args: "arg0" arg1"
```

Comment:
Comments are not completed properly. Should be `"arg0" "arg1"` or `"arg0 arg1"` (I'm not sure which).

Reply:
The args example needs to be something different.